src/smtpproto/protocol.py

```diff
@@ -59,6 +59,7 @@ class SMTPClientProtocol:
     """The (E)SMTP protocol state machine."""
 
     _state: ClientState = field(init=False, default=ClientState.greeting_expected)
+    _smtputf8_message: bool = field(init=False, default=False)
```

Comment:
Should be consistent with the default value in the mail command?
```suggestion
    _smtputf8_message: bool = field(init=False, default=True)
```